chef_master/source/dk_windows.rst

```diff
@@ -1,101 +1,195 @@
-   At line:1 char:1
-   + "chef shell-init powershell | Invoke-Expression" >> $PROFILE
-   + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-       + CategoryInfo          : OpenError: (:) [Out-File], DirectoryNotFoundException
-       + FullyQualifiedErrorId : FileOpenFailure,Microsoft.PowerShell.Commands.OutFileCommand
+   Set-ExecutionPolicy RemoteSigned
 
-In this situation, run the following ``chef shell-init`` command instead:
+Create a Profile
++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. code-block:: bash
+Once the execution policy is in place, initialize the Windows environment with the following commands: 
 
-   if(Test-Path $PROFILE){ chef shell-init powershell | Add-Content $PROFILE } else { New-Item -Force -ItemType File $PROFILE; chef shell-init powershell | Add-Content $PROFILE }
+Create a ``$PROFILE`` file if one doesn’t exist already:
 
-.. end_tag
+.. code-block:: shell
 
-Set the Execution Policy
------------------------------------------------------
-.. tag ruby_set_system_ruby_as_chefdk_ruby_windows_admin
+   PS C:\Users\<username>> if(!(Test-Path $PROFILE)) { New-Item -Force -ItemType File $PROFILE }
 
-On new Windows machines, PowerShell scripts will not work until an administrator runs the following command:
+Next, add the output of ``chef shell-init powershell`` to your ``$PROFILE`` (This command does not print the output onto the screen):
 
-.. code-block:: bash
+.. code-block:: shell
 
-   Set-ExecutionPolicy RemoteSigned
+   PS C:\Users\<username>> chef shell-init powershell | Add-Content $PROFILE
 
-.. end_tag
+Once initialized, your paths and environment variables are set up correctly for every new PowerShell session. See your environment settings by using ``cat $PROFILE``:
+
+.. code-block:: shell
+
+    PS C:\Users|<username>> cat $PROFILE
+    $env:PATH="C:/opscode/chefdk/bin;C:/Users/<username>/AppData/Local/chefdk/gem/ruby/2.5.0/bin;C:/opscode/chefdk/embedded/bin;C:/opscode/chefdk/embedded/git/usr/bin"
+    $env:GEM_ROOT="C:/opscode/chefdk/embedded/lib/ruby/gems/2.5.0"
+    $env:GEM_HOME="C:/Users/<username>/AppData/Local/chefdk/gem/ruby/2.5.0"
+    $env:GEM_PATH="C:/Users/<username>/AppData/Local/chefdk/gem/ruby/2.5.0;C:/opscode/chefdk/embedded/lib/ruby/gems/2.5.0"
 
 Spaces and Directories
------------------------------------------------------
++++++++++++++++++++++++++++++++++++++++++++++++++++++
 .. tag windows_spaces_and_directories
 
-Directories that are used by Chef on Windows cannot have spaces. For example, ``C:\Users\Steven Danno`` will not work, but ``C:\Users\StevenDanno`` will. Because of this, the ``knife supermarket install`` subcommand will fail if the directory contains a space.
+Directories that are used by Chef on Windows cannot have spaces. For example, ``C:\Users\User Name`` will not work, but ``C:\Users\UserName`` will. Because of this, the ``knife supermarket install`` subcommand will fail if the directory contains a space.
```

Comment:
This section appears to be duplicated elsewhere (I commented on it further down this PR)